### PR TITLE
feat: add exit hook to cancel order when process terminated by user

### PR DESCRIPTION
**Pull request by rogerwatkins**
order is not cancelled if already filled.

Ran following tests:
PLACE A STOP-LIMIT ORDER -> CANCEL - PASS
PLACE A STOP-LIMIT ORDER -> TRIGGER -> OCO - PASS
PLACE A LIMIT ORDER -> CANCEL - PASS
PLACE A LIMIT ORDER -> TRIGGER -> OCO - PASS

### CODE REVIEW

File: src/cli.ts [hunk 1]
```diff
@@ -86,6 +86,20 @@ const {
 
 const debug = require("debug")("binance-oco");
 
+const exitHooks = (cancel: Function) => {
```
Comment (tony-ho) [inline src/cli.ts]: Is it possible to implement this logic using an existing node package such as [exit-hook](https://www.npmjs.com/package/exit-hook) or [node-cleanup](https://www.npmjs.com/package/node-cleanup)?

**Comment by rogerwatkins** [comment]
I'll take a look at the ESLint and prettier warnings and update the PR.

File: src/cli.ts (hunk 1 above)
Comment (rogerwatkins) [reply src/cli.ts]: Did not know about those - will investigate!

**Comment by rogerwatkins** [comment]
TSC Broke with async-exit-hook library - needs custom tsc declaration.

**Pull request merged**